tests/kernel/gen_isr_table/src/main.c

```diff
@@ -381,4 +381,81 @@ static void *gen_isr_table_setup(void)
 	return NULL;
 }
 
+#ifdef CONFIG_MULTI_LEVEL_INTERRUPTS
+ZTEST(gen_isr_table, test_multi_level_bit_masks_sec)
+{
+#if CONFIG_1ST_LEVEL_INTERRUPT_BITS < 10 && CONFIG_2ND_LEVEL_INTERRUPT_BITS < 10
+	ztest_test_skip();
+#endif
+
+	unsigned int irq = 0x400;
```

Comment:
the tests are a little magic number-y. Is it possible to add some comments or derive constants from e.g. Kconfig values?

Reply:
I admittedly just selected random values that qould qualify as the different level irqs.